Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -309,8 +315,15 @@ def fetch_incidents(client: Client, max_results: int, last_run: Dict[str, Union[
         start_time=last_fetch
     )
 
+    if reset_mirror:
+        integration_context, version = get_integration_context_with_version()
+        incident_mirror_reset: str = ','.join([incident['id'] for incident in incidents])
+        demisto.debug(f'Adding incidents id to mirror reset set:{incident_mirror_reset}')
+        integration_context['XSOARMirror_mirror_reset'] = incident_mirror_reset
+        set_integration_context(context=integration_context, version=version)
```

Comment:
wouldn't it be easier to work with a `dict` rather an `CSV`?

Reply:
not sure about the key value here.
I tried working with a set and a list but context data worked for me just with a string.

Reply:
changed to dict.